staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.pb.go

```diff
@@ -5270,14 +5270,14 @@ func (m *ImageStatusResponse) GetInfo() map[string]string {
 // AuthConfig contains authorization information for connecting to a registry.
```

Comment:
can we link to the datapolicy kep here and/or provide a brief explanation of any client and server processing requirements?  I believe the expectation here is these fields if output to a log would cause the log entry to be filterable by some datapolicy filtering tool. What's not clear is if there is any new expectation (or not) that container runtime integrations must apply a filter/mask for these fields (remove them from the logs replace with xxx etc.). I'd have to check to make sure but I believe I remember seeing commits over time that removed these fields from the logs.